src/components/QuestionsList.tsx

```diff
@@ -0,0 +1,59 @@
+'use client';
+import * as React from 'react';
+import { Box, Typography, Accordion, AccordionSummary, AccordionDetails } from '@mui/material'; 
+import { CaretDownOutlined } from '@ant-design/icons';
+import { SyntheticEvent, useState } from 'react';
+import { faqData } from '../constants/faqData';
+import { useTheme } from '@mui/material/styles';
+
+export default function QuestionsList() {
+  const [expanded, setExpanded] = useState<string | false>(false);
```

Comment:
```suggestion
  const [expanded, setExpanded] = useState<string>("");
```
You can do some research about falsy values